newIDE/app/src/MainFrame/index.js

```diff
@@ -278,13 +281,21 @@ const initialPreviewState: PreviewState = {
   overridenPreviewExternalLayoutName: null,
 };
 
+// Simpler version of the CaptureOptions, as only the timing is needed to start configuring the preview capture.
+type LaunchCaptureOptions = {|
+  screenshots: Array<{|
+    timing: number,
```

Comment:
at the moment, the only config for a screenshot is the timing after the preview starts.
It leads to OK results, but maybe we want something more fine-tuned, like a screenshot of each scene ? but then it becomes hard to categorize them well in the backend, as it's just a list of screenshots.